### DISCUSSION TITLE
Hows it coming along?

### DISCUSSION THREAD

**Discussion by mikeoptics** [twbs/bootstrap#31030] [General] [answered]
Just about to start developing a new website and didn't want to get half way and find out v5 is released. 
I know these things take as long as they need but do you think it will be coming soon enough for me to wait?

Thanks for all you do, been using Bootstrap for years.

**Comment by techboyg5** [comment, marked as answer]
Bootstrap 4 (currently 4.5.0) would be the best choice both right now and a few months after Bootstrap 5 comes out. You shouldn't use something right after it comes out as it can have bugs.

**Reply by mikeoptics** [reply]
Thank you. Il do just that. Next project perhaps.
